internal/api/v1beta1/relation.go

```diff
@@ -65,6 +66,13 @@ func (h Handler) CreateRelation(ctx context.Context, request *shieldv1beta1.Crea
 		return nil, grpcBadBodyError
 	}
 
+	if !namespace.IsSystemNamespaceID(request.GetBody().ObjectNamespace) {
+		_, err := h.resourceService.Get(ctx, request.GetBody().GetObjectId())
```

Comment:
What is the purpose of doing this?

should we use the resource id from this get for the flow below?

Reply:
Relation creation happens for system-namespace entities and regular resources.

For regular resources
- We want to check if the ObjectId sent (which can be in the format of uuid but still can be a resource name - case of guardian appeal), is a valid resource id or not.

For system namespace
- we don't need any extra check in the exisiting flow

`should we use the resource id from this get for the flow below?`
we are not checking this for system ns, so we can't use it below